backend/tests/test_api.py

```diff
@@ -2,3 +2,62 @@ def test_hello(client):
     response = client.get("/")
     assert response.status_code == 200
     assert response.json() == {"message": "Hello user! Tip: open /docs or /redoc for documentation"}
+
+
+def test_ping(client):
+    response = client.get("/ping")
+    assert response.status_code == 200
+    assert response.json().get("ping").startswith("pong @")
+
+
+def test_import_from_github(client):
+    response = client.post(
+        "/test/testing_branch/import",
+        json={"description": "test_description"},
+    )
+    assert response.status_code == 200
+    assert response.json() is True
```

Comment:
do this test really use Github ?

I'm not keen on test depending on external servers, we should better mock the github functions (the one from the library).